activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -322,6 +322,29 @@ def create_table(table_name, id: :primary_key, primary_key: nil, force: nil, **o
           schema_cache.clear_data_source_cache!(table_name.to_s)
         end
 
```

Comment:
FYI I’ll remove all my comments here; they’re just notes to myself as I work. It’s probably better to just skip over them.